.travis/travis-coverity.sh

```diff
@@ -24,7 +24,7 @@ trap coverity_finalize EXIT
 # Always run the Coverity scan on the oldest supported Fedora
 # because it generally lags behind with GCC compatibility.
 MMD_OS=fedora
-MMD_RELEASE=30
+MMD_RELEASE=31
 MMD_IMAGE=fedora/fedora:${MMD_RELEASE}-$(uname -m)
```

Comment:
ACK. But since doing so involves more than another 1-line change to this file, I'm going to see what I can do to enhance `mmd_run_docker_tests()` in `travis-common.inc` so it can be used to run the scan and eliminate all the duplication here.

Reply:
@sgallagher Is it OK if split off the previously mentioned enhancement into a separate follow-up PR?

Reply:
Yeah, that probably makes sense.